src/DotNetWorker.Grpc/GrpcCollectionModelBindingData.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc.Messages
+{
+    internal class GrpcCollectionModelBindingData : Microsoft.Azure.Functions.Worker.Core.CollectionModelBindingData
+    {
+        private readonly CollectionModelBindingData _modelBindingDataArray;
```

Comment:
Do we need this private variable? `ModelBindingDataArray` is get only prop and we can set the value of that inside the constructor.